reactive/openjdk.sh

```diff
@@ -32,6 +32,17 @@ function install() {
     local java_type=$(config-get 'java-type')
     local java_major=$(config-get 'java-major')
 
+    if grep -q trusty /etc/lsb-release; then
+        # ensure that Java 8 can be installed on trusty
+        add-apt-repository -y ppa:openjdk-r/ppa
+        apt-get update -qq
+    fi
+
+    if [[ `uname -p` == 'ppc64'* ]]; then
```

Comment:
This looks weird as all get out to me, and I'm not near a prompt on a ppc box to verify. To earn the prestigious +1 you so desire, prove to me that the weird 'ppc64'* string does what you want it to do, that is, match ppc64le.